applications/Preprocessor/lib/output_impl.h

```diff
@@ -72,6 +72,7 @@ void printOtherEntities(std::ofstream& output, const Mesh<dimension>& mesh,
                         tag<d>) {
     for (auto entity : mesh.template getEntities<d>()) {
         std::set<std::size_t> localPartitions;
+        output << "codim" << dimension - d << '\n';
```

Comment:
Codim is now repeated for every edge/face.

```suggestion
    output << "codim" << dimension - d << '\n';
    for (auto entity : mesh.template getEntities<d>()) {
        std::set<std::size_t> localPartitions;
```